String/union.py

```diff
@@ -0,0 +1,18 @@
+def union(list1,list2):
```

Comment:
Follow proper naming, list1 and list2 can be confusing for people. Try to use, A and B or something like that.